modules/addBookFormHandler.js

```diff
@@ -0,0 +1,32 @@
+import * as elements from './elements.js';
+import Book from './books.js';
+import spaNav from './spaNav.js';
+import BooksList from './booksList.js';
+
+export default elements.form.addEventListener('submit', async (e) => {
+  e.preventDefault();
+
+  const msg = document.querySelector('.error-message');
+  function clear() { msg.innerHTML = ''; }
+
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  const books = JSON.parse(localStorage.getItem('books')) || [];
```

Comment:
- [ ] **Optional:** This code is repeated as it already exists in the `Book.getBooks()` class function. You may replace it so with function.